project/justfix_environment.py

```diff
@@ -358,6 +358,11 @@ class JustfixEnvironment(typed_environ.BaseEnvironment):
     # behavior.
     IS_DEMO_DEPLOYMENT: bool = False
 
+    # Whether or not the Eviction Free NY site is "suspended," meaning
+    # that users can visit the site but are blocked from starting a
+    # new hardship declaration form or completing one in progress.
+    IS_EFNY_SUSPENDED: bool = False
```

Comment:
This looks great! I was looking into best practices for feature flags (https://thysniu.medium.com/coding-with-feature-flags-how-to-guide-and-best-practices-3f9637f51265), and I think putting them in as environment variables is a perfectly fine idea for now. Are you thinking we'll leave this feature flag in here in case EFNY is suspended in the future, or that we'll delete it once we launch it?

Reply:
If you're thinking we might want to leave it in for longer, and we might start to have more feature flags, we might look into a solution that doesn't result in a long list of extra env variables.

One option is https://cfpb.github.io/django-flags/ (you know I love anything by the CFPB), but it's a Django app to handle feature flags for you.

Reply:
Ooooo ok that's great to know about! yeah we've sort've just gotten accustomed to using ENV vars as feature flags but I agree that it can get annoying if we have many of them. LEt's keep this in mind.

Reply:
And unfortunately, it is very possible that the tool could need a sudden suspension of there is a challenge to the new law, so I do think we need to keep this flag here permanently, or at least for the foreseeable future.

Reply:
ok! yeah, i think this is a fine way of implementing it for now.